app/javascript/src/components/Dashboard/Notes/NoteItem/NoteItem.jsx

```diff
@@ -45,9 +45,12 @@ const NoteItem = ({
         />
         <div className="flex items-center">
           <Clock color="#1e1e20" size={22} />
-          <Tooltip content={formatDate(createdAt)} position="bottom-end">
+          <Tooltip
+            content={formatDateWithDayAndTime(createdAt)}
+            position="bottom-end"
+          >
             <Typography className="mx-2" style="body3">
-              {lastUpdated(createdAt)}
+              {createdDateTillNow(createdAt)}
```

Comment:
I believe the util function needn't be tied up with just the `created_at` entity. Any date can be manipulated by the util function. Also I believe the following is a better name that conveys what's happening.
```suggestion
              {formatDateRelativeToNow(createdAt)}
```